browser/src/view/components/Skeleton/Skeleton.scss

```diff
@@ -0,0 +1,20 @@
+@use '../../../styles/common/variables' as *;
```

Comment:
I approved but this file name shouldn't be cap'd.  Capitalization is only used for classes/class-like things.  Files exporting react components are capitlaized out of convention at this point because they all used to be classes.

Reply:
Updated the file names